docs/tests/core-concepts/client/plans/file_export.cue

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"dagger.io/dagger"
+	"dagger.io/dagger/core"
+
+	"universe.dagger.io/go"
+)
+
+dagger.#Plan & {
+	client: filesystem: output: write: contents: actions.buildhello.output
+
+	actions: buildhello: {
+		_source: core.#WriteFile & {
+			input: dagger.#Scratch
+			path:  "/helloworld.go"
+			contents: #"""
+				package main
+				import "fmt"
+				func main() {
+				  fmt.Println("Hello, World!")
+				}
+				"""#
```

Comment:
nit: Remove `#` ([Avoid raw strings](https://docs.dagger.io/1226/coding-style#avoid-raw-strings)).

Reply:
Ah I didn't realize the distinction until now, thanks! Fixed it